index.html

```diff
@@ -6,9 +6,47 @@
     <link rel="icon" type="image/x-icon" href="/images/favicon.svg" />
     <link rel="stylesheet" href="styles/style.css" />
     <!-- Fontlar buraya gelecek -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
     <title>Grover</title>
+
   </head>
+  
+  
   <body>
-    <!-- Kodlar buraya gelecek -->
+     <div class="main">
+      <span class="G-Logo">
+        <img src="images/Header/favicon.svg" alt="G logo">
+        <h1 class="title-Grover">Grover</h1>
+      </span>
+
+      <span class="header-button">
+        <button class="button-menu">Home</button>
+        <button class="button-menu">Products</button>
+        <button class="button-menu">Service</button>
+        <button class="button-menu">Shop</button>
+      </span>
```

Comment:
Menü için <nav> <ul> ve <li> etiketlerini kullan

Reply:
Dikkat edildi